vnet/lib/vnet/endpoints/1.0/datapaths.rb

```diff
@@ -41,16 +41,21 @@ def self.put_post_shared_params
   end
 
   param_uuid M::Interface, :interface_uuid, required: true
-  param :broadcast_mac_address, :String, required: true, transform: PARSE_MAC
+
+  # 'broadcast_mac_address' is deprecated and should be phased out.
+  param :mac_address, :String, transform: PARSE_MAC
```

Comment:
I see. I didn't realize it wasn't necessary for a single host. Then that means it's indeed ok to not have the required flag set.

One thing I'm worried about though, is that people will create single host networks and then get stuck when trying to move them to multiple hosts. We should make sure to document this properly. Currently the installation guide has only one host so a guide to add more hosts to it would make sense. That's of course a different issue.